qa/tasks/ses_qa.py

```diff
@@ -163,10 +170,20 @@ def drive_replace_check(self, **kwargs):
         Replaced osd_id should be back in the osd tree once stage.3 is ran
         """
         total_osds = self.master_remote.sh(number_of_osds_in_cluster)
-        assert int(total_osds) == 4, "Unexpected number of osds {} (expected 4)".format(total_osds)
+        disks = self._get_drive_group_limit()
+        assert int(total_osds) == int(disks), "Unexpected number of osds {} (expected {})".format(total_osds, disks)
```

Comment:
(nit) Isn't assert displays numbers already, so no need to provide them in the message, like, "Unexpected number of osds" would be enough.

Reply:
Nope:
```
Python 2.7.16 (default, Mar 04 2019, 07:13:50) [GCC] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> total_osds=4
>>> disks=5
>>> assert total_osds == disks, "Unexpected number of osds {} (expected {})".format(total_osds, disks)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: Unexpected number of osds 4 (expected 5)
>>> assert total_osds == disks, "Unexpected number of osds"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: Unexpected number of osds
```